freva-rest/src/freva_rest/auth.py

```diff
@@ -0,0 +1,212 @@
+"""Definition of routes for authentication."""
+
+import os
+from typing import Annotated, Optional
+
+import aiohttp
+import jwt
+from fastapi import Depends, Form, HTTPException, status
+from fastapi.security import (
+    OAuth2AuthorizationCodeBearer,
+    OAuth2PasswordRequestForm,
+)
+from pydantic import BaseModel
+
+from .logger import logger
+from .rest import app, server_config
+
+TIMEOUT: aiohttp.ClientTimeout = aiohttp.ClientTimeout(total=5)
```

Comment:
if 5 is not intended, what do u think to use env variable to avoid hard coding here ...

Reply:
In general my view on config variables is: the less the merrier ;). Just have a look at freva-web and you'll understand what I mean.